lib/i18n/tasks/translators/deepl_translator.rb

```diff
@@ -60,11 +69,24 @@ def no_results_error_message
 
     private
 
-    # Convert 'es-ES' to 'ES'
-    def to_deepl_compatible_locale(locale)
+    # Convert 'es-ES' to 'ES', en-us to EN
+    def to_deepl_source_locale(locale)
       locale.to_s.split('-', 2).first.upcase
     end
 
+    # Convert 'es-ES' to 'ES' but skip specific locales
+    # This and the above should probably be done by the deepl gem
+    def to_deepl_target_locale(locale)
+      loc, sub = locale.to_s.split('-')
+      if SPECIFIC_TARGETS.include?(loc)
+        fail ::I18n::Tasks::CommandError, I18n.t('i18n_tasks.deepl_translate.errors.specific_target_missing') unless sub
```

Comment:
The API docs say that the the language without a region is supported for backwards compatibility. Perhaps we should issue a warning instead?

Reply:
yes that may be a bit nicer, i'll convert that to a warn_deprecated, as thats what i found in other regions of the gem.